openapi.yaml

```diff
@@ -16431,31 +16431,36 @@ components:
         name:
           type: string
           description: |
-            The name of this Record. For requests, this property's actual usage and whether it is required depends on the type of record this represents:
+            The name of this Record. For requests, this property's actual usage and whether it is required depends
+            on the type of record this represents:
 
             `A` and `AAAA`: The hostname or FQDN of the Record.
 
             `NS`: The subdomain, if any, to use with the Domain of the Record.
 
-            `MX`: The subdomain.
+            `MX`: The mail server subdomain. To create a [Null MX record](https://datatracker.ietf.org/doc/html/rfc7505),
```

Comment:
May I suggest changing the order of these instructions? I would maybe say something like:

```suggestion
            `MX`: The mail server subdomain. To create a [Null MX record](https://datatracker.ietf.org/doc/html/rfc7505), first remove any additional MX records, then enter an empty string for this property.
```

With the appropriate links and parenthetical info, etc.